README.md

```diff
@@ -86,20 +85,20 @@ path:
 
 ``` puppet
 class site::nfs_server (
-  $usekrb5 = hiera('nfs::simp_krb5',false),
-  $client_nets = defined('$::client_nets') ? { true => $::client_nets, default => hiera('client_nets') }
+  $kerberos = simplib::lookup('simp_options::kerberos', { 'default_value' => false, 'value_type' => Boolean }),
+  $trusted_nets = defined('$::trusted_nets') ? { true => $::trusted_nets, default => hiera('trusted_nets') }
```

Comment:
yes, but I basically sed-ed the readme. I'm waiting to do full reconstructs in SIMP-2213.